--- origsrc/kvkbd-0.6/src/CMakeLists.txt	2008-10-11 07:28:19.000000000 -0500
+++ src/kvkbd-0.6/src/CMakeLists.txt	2010-01-10 21:22:37.573561900 -0600
@@ -22,7 +22,7 @@ QT4_ADD_DBUS_ADAPTOR(kvkbd_SRCS org.kde.
 
 kde4_add_executable(kvkbd ${kvkbd_SRCS})
 
-target_link_libraries(kvkbd ${KDE4_KDECORE_LIBS} Xtst ${KDE4_KDEUI_LIBS})
+target_link_libraries(kvkbd ${KDE4_KDECORE_LIBS} ${KDE4_KDEUI_LIBS} ${X11_XTest_LIB} ${X11_X11_LIB})
 
 install(TARGETS kvkbd ${INSTALL_TARGETS_DEFAULT_ARGS})
 
